src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h

```diff
@@ -1,7 +1,9 @@
 #define DEVICE_NAME "DIY2400 E28"
 
 // Any device features
-#define USE_OLED_SPI
```

Comment:
I would not change default value. Maybe OLED type into user defines for DIY purposes?